ow_plexil/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ generate_messages(
 
 catkin_package(
   LIBRARIES ${PROJECT_NAME}
-  CATKIN_DEPENDS roscpp roslib rospy ow_lander actionlib_msgs geometry_msgs ow_faults
+  CATKIN_DEPENDS roscpp roslib rospy ow_lander actionlib_msgs geometry_msgs ow_faults_injection
```

Comment:
@Samahu pointed this out in the ow_simulator PR.  I think that ow_faults_detection should be a dependency of autonomy, but not ow_faults_injection.